diamm/serializers/website/composition.py

```diff
@@ -36,6 +38,12 @@ class Meta:
 class CompositionDetailSerializer(serializers.HyperlinkedModelSerializer):
     composers = CompositionComposerSerializer(many=True)
     sources = CompositionSourceSerializer(many=True)
+    type = serializers.SerializerMethodField()
+
     class Meta:
         model = Composition
-        fields = ('url', 'title', 'composers', 'sources')
+        fields = ('url', 'title', 'composers', 'sources', 'pk', 'type')
+
+    def get_type(self, obj):
+        return self.Meta.model.__name__.lower()
```

Comment:
For example, you could file an issue for this one -- 'rewrite Composition Detail Serializer to serpy'...